SE-17499: Clarification for Maven plugin usage in Studio. (#166)
mulesoft/docs-munit: modules/ROOT/pages/munit-maven-plugin.adoc

:keywords: munit, testing, unit testing
:page-aliases: munit-maven-support.adoc, to-set-up-munit-maven-plugin.adoc, faq-working-with-parent-pom.adoc, to-inherit-parent-plugin.adoc, to-ignore-parent-plugin.adoc, to-override-parent-plugin.adoc, munit-maven-plugin-configuration.adoc,to-configure-munit-maven-plugin-maven.adoc

MUnit provides a Maven plugin that enables you to run your MUnit tests as part of your continuous integration environment.

> The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation only apply when running Munit through Maven.

Reviewer: ```suggestion
The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation apply only when running MUnit through Maven.
```

| `runtimeVersion`
| String
| Version of the Mule runtime +

> When running your tests from Maven, this value is taken from the `minMuleVersion` parameter in the `mule-artifact.json` file. +

Reviewer: ```suggestion
When you run tests from Maven, this value is taken from the `minMuleVersion` parameter in the `mule-artifact.json` file. +
```

Reviewer: otherwise it says that "this value" runs the test
